controller_utils/precice_struct/PS_ParticipantSolver.py

```diff
@@ -29,11 +29,14 @@ class SolverNature(Enum):
     TRANSIENT = 1
 
 class PS_ParticipantSolver(object):
-    """Class to represent a participat in the preCICE data structure """
+    """Class to represent a participant in the preCICE data structure """
+
+    dim: SolverDimension
+    dimensionality: int
 
     # TODO: one solver might have more than one couplings!!!
 
-    def __init__(self, participant: UI_Participant, coupling: UI_Coupling, conf ): # conf:PS_PreCICEConfig
+    def __init__(self, participant: UI_Participant, coupling: UI_Coupling, conf): # conf:PS_PreCICEConfig
```

Comment:
@Toddelismyname @VanLaareN `coupling` and `conf` are never used. This looks like an error to me.

Reply:
How should I handle this? Remove the parameters or are they actually needed?